_docs/_latest/setup/forseti-on-gke.md

```diff
@@ -220,6 +220,53 @@ helm upgrade -i forseti forseti-security/forseti-security \
     --set-string serverConfigContents="$(gsutil cat gs://<BUCKET_NAME>/configs/forseti_conf_server.yaml | base64 -)" \
     --values=forseti-values.yaml
 ```
+
+### Deploying with config-validator on GKE
+
+The config-validator in Forseti on-GKE obtains policies from a policy-library in a Git repository via SSH.  The pre-requisites for this are as follows.
```

Comment:
Remove dash in `Forseti on-GKE` --> `Forseti on GKE`

Reply:
I am fairly certain this is grammatically correct.  "on-GKE" is a compound adjective so it should be hyphenated.

Reply:
Then we should correct any "on GKE" that are in this file to be consistent

Reply:
What is the difference between Forseti-on-GKE and Forseti on-GKE?

Reply:
Forseti is a noun, on-GKE is modifying that noun.

Reply:
Ok- thanks for the clarification, Ken!